sound/soc/sof/topology.c

```diff
@@ -166,6 +177,11 @@ static int sof_keyword_dapm_event(struct snd_soc_dapm_widget *w,
 				swidget->widget->name);
 		break;
 	case SND_SOC_DAPM_POST_PMD:
+		if (spcm->stream[stream].d0i3_active) {
+			dev_dbg(sdev->dev, " KWD will work in suspended, don't suspend it\n");
```

Comment:
Minor nit: "KWD stream will be active in D0ix, not suspending"

Reply:
Thanks @kv2019i , but @plbossart mentioned several times that wording "active" could be confusing also, I am not a native speaker, but I like your wording here, let me consider do some changes there.
And, I will change the flag to ignore_dapm_event at the same time.